docs/automation-rules.rst

```diff
@@ -69,6 +72,12 @@ Currently, the following actions are available:
    The stable version is also automatically updated at the same time.
    See more in :doc:`versions`.
 
+.. note::
+
+   Default versions aren't deleted even if they match a rule.
+   You can use the ``Set version as default`` action to change the default version
+   before deleting the current one.
```

Comment:
I'd move this note above the previous one to keep it close to "Delete version". Even, if possible, I'd put it inside the bullet.

Otherwise, it's not easy to realize that this note is strictly related to "Delete version" action.